docker/README.md

```diff
@@ -0,0 +1,54 @@
+# Instructions for building the Docker container from the Dockerfile and running code within the container
+
+## Starting docker daemon within WSL2 or Linux OS
+`sudo service docker start`
+
+`sudo dockerd > /dev/null 2>&1 &`
+
+
+## Building an image from the Dockerfile
```

Comment:
Most of the time we shouldn't have to do a `docker build` right? Once the image is built and pushed to the container registry, we should just be able to do a `docker pull.` I think it'd be worth saying that here. What do you think?

Reply:
Yes, that's right. I'll add a comment about that here